runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/actions/image.ts

```diff
@@ -0,0 +1,10 @@
+import englishImageActionsTranslations from '#src-app/translations/en/process/actions/image';
+
+const imageActionsTranslations: typeof englishImageActionsTranslations = {
+    'Process.Details.Modeler.Actions.Image.Grayscale.Label': 'Przekowertuj do skali szarości',
+    'Process.Details.Modeler.Actions.Image.Grayscale.ImagePath': 'Ścieżka do obrazu',
+    'Process.Details.Modeler.Actions.Image.Grayscale.ConvertedImagePath': 'Ścieżka przekonwertowanego obrazu',
+    'Process.Details.Modeler.Actions.Image.Grayscale.ConvertedImageAbsolutePath': 'Ścieżka absolutna przekonwertowanego obrazu',
+};
```

Comment:
add tooltip: If you are not MIkołaj Szymczak do not use because nobody else nows how it works